torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5913,19 +5967,22 @@ def map_arg(arg):
         def maybe_non_contig(tensor):
             return tensor if not non_contiguous else make_non_contiguous(tensor)
 
+        def maybe_conj(tensor):
+            return tensor if not dtype.is_complex else tensor.conj()
```

Comment:
This seems a little helter skelter to me: `maybe_non_contig` toggles contiguity based on the `non_contiguous` parameter to the test itself (so you can trigger contig versus non contig). But `maybe_conj` operates based on if the dtype is complex... which doesn't actually do anything because `tensor.conj()` is a noop anyway of the tensor isn't complex.  For the purposes of exercising conjugate views this is fine, because now everything is conjugated, but it's suboptimal for test coverage, since all the non-conjugate view paths are now untested.

Reply:
True although it's only a matter of time before the method_tests would be removed + we are not adding any new method_tests, so I think it's an ok workaround